### ISSUE TITLE
Feedback: Regression

### ISSUE THREAD

**Issue by cabergh** [whitead/dmol-book#83]
- In 3.2 Overfitting: “We will split our data into two: training data and testing data.” -> “...into two subsets: one with training data and one with test data…”.

- In 3.2. Overfitting: “Typically this is done with an 80%/20%, so that you train on 80% of your data.” could be clearer, e.g. “...80/20 split, so you train on 80% of the data and test on the remaining 20%”. It might also be good to mention how to select the split fraction, e.g. often you need to focus your limited data on training rather than testing.

- In 3.2 Overfitting: “Let’s begin by seeing what effect the split of train/data”. Should it be “train/test data” here?

- In 3.2 Overfitting: “...the same as the previous chapter” -> “...as in the previous chapter”

- In the note about the decorator, I believe Python is spelled with capital P.

- In 3.2 Overfitting: “we can accidentally learn approximate the sum” -> “...learn to approximate…”

- In 3.2 Overfitting: “Would there overfitting with fewer features that are perfectly correlated with labels?” -> “Would there be overfitting…”

- In 3.3 Exploring Effect of Feature Number: I think this title could be formulated better, e.g. as “Exploring the Effect of the Number of Features”

- In 3.3 Exploring Effect of Feature Number: “...effect the feature number has on the data.” I think it’s better to write “number of features” rather than “feature number”?

- In 3.3 Exploring Effect of Feature Number: “In the results below, they are averaged over possible features and possible training data splits to deal with this.” This sentence is a bit unclear. What do “they” and “this” refer to? 

- In 3.3 Exploring Effect of Feature Number: “...but I’ve omitted it for simplicity.” -> “...omitted it here…”.

- In 3.3 Exploring Effect of Feature Number: “...requires 100 more times data…” -> “...requires 100 times more data…”.

- In 3.4 Bias Variance Decomposition: I’m not sure what is meant by “However, bias does not determine the expected error on an unseen data point alone, there other terms”. Is it supposed to be “...since there are other terms to consider as well.”?

- In 3.4 Bias Variance Decomposition: “Noise is set by your data and not controllable.” -> “...and is not controllable.” 

- Note in 3.4: “The decomposition above does not imply it.” What does “it” refer to?

- In 3.6.1 k-Fold Cross-Validation: “...testing error depends on size of train/test split” -> “...depends on the size of the train/test split.”

- In 3.6.1 k-Fold Cross-Validation: “Let’s now use k-fold cross in two examples” -> “...cross-validation...”?

- In 3.6.1 k-Fold Cross-Validation: “Rather than use gradient...” -> “Rather than using…”

- In 3.6.1 k-Fold Cross-Validation: “Now what effect does k have on this process?” Which process?

- Side comment in 3.7.1: “Classically bootstrap resampling and jacknife, its predecessor, are for estimating variance in model parameters (i.e., model variance).” could be better formulated. Maybe “...can be used for estimating…”? Also, shouldn't terms be written in boldface as previously done in this book?

- In 3.7.1. Bootstrap Resampling: “...as many as desired new training datasets that are the same size as the original…” -> “...as many new equal-in-size/equisized training datasets as desired…”.

- In 3.7.1. Bootstrap Resampling: “...our new dataset has less members…” -> “...has fewer members…”. “...makes-up...” -> “...makes up…”.

- In 3.7.1. Bootstrap Resampling: “If your training dataset originally has data A, B, C, D, E, our bootstrap resampled training data is:” I think this sentence is slightly misleading, since it makes it sound like we always get the written configurations. Change to “...training data might be:”?

- In 3.7.1. Bootstrap Resampling: “...you could choose a fewer number…” -> “...lower number…”

- In 3.7.1. Bootstrap Resampling: “...approach to get test error.” -> “...to get a test error.”

- In 3.7.1. Bootstrap Resampling: “...this is bit overly complex…” -> “...this is a bit…”

- In 3.7.1. Bootstrap Resampling: “It also under-estimates the error, because it has no estimate of the test error. It only accounts for variation due to training data. Using the language above, it only accounts for model variance but not model bias.” This part would be clearer of it was clarified what “it” means.

- In 3.7.1. Bootstrap Resampling: “Jacknife+ carriers strong guarantees...” -> “...carries…”

- Side comment in 3.7.1: I believe Jacknife should be in bold to follow the previous format.

- In 3.8.1 Leave one class out cross-validation: First the title should be capitalized to follow the previous format. Then, I don’t quite follow this sentece: “The second challenge, that you’re leaving your training data due to how points are generated, can be assessed by computing a more realistic estimate of model error.”

**Comment by awoonor** [comment]
Section 3.3 
Paragraph 1: "We'll return the solubility dataset to discuss this." should read "We'll return **to** the solubility dataset to discuss this."

Final Paragraph: "... reducing noise by a factor of 10 requires 100 more times data" should read "... reducing noise by a factor of 10 requires 100 **times more** data".
